graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/FederationDirectives.java

```diff
@@ -69,18 +69,21 @@ private static GraphQLArgument fieldsArgument(String value) {
             .name(keyName)
             .validLocations(OBJECT, INTERFACE)
             .argument(fieldsArgument)
+            .repeatable(true)
```

Comment:
In the comment of
```java
/* directive @key(fields: _FieldSet!) on OBJECT | INTERFACE */
```
above, can you add `repeatable` there?

Reply:
ok will do!